### PR TITLE
move to current (am5f8d0r1) am5 code

### PR DESCRIPTION
**Pull request by uramirez8707**
The executable generated from fre/canopy reproduced the executable generated from fre/bronx-21 for a c96 amip experiment!

### CODE REVIEW

**Review by singhd789 — APPROVED** [review]

File: am5-compile/compile.yaml [hunk 1]
```diff
@@ -93,13 +94,29 @@ compile:
...
+
+              cd src/MOM6
+              git checkout 878fd1ef671456a804c598df606e4cf7c803c203
+
+              cd ../SIS2
+              git checkout 00c3b5be13721a7af99f62f2163969e8dd303df7
+
+              cd ../icebergs
+              git checkout 6a8944fc7544608942de9a42bd3b96f6e56684e0
+              cd ../../../..
```
Comment (ceblanton) [inline am5-compile/compile.yaml:115]: how about setting the `MOM6_GIT_TAG` property in the am5.yaml, as

```
fre_properties:
    &MOM6_GIT_TAG        "foo"
```

and then using it here as

```
              cd src/MOM6
              git checkout *MOM6_GIT_TAG
```

**Review by ceblanton — APPROVED** [review]
Nice! In the future we can store the mom6_git_tag as a yaml variable, probably, just like we used it as a FRE property.

File: am5-compile/compile.yaml (hunk 1 above)
Comment (uramirez8707) [reply am5-compile/compile.yaml:115]: This does not seem to work:
This is checkout script that is generated: /ncrc/home1/Uriel.Ramirez/fremake_canopy/test/am5f8d0r1_compile/src/checkout.sh
```
cd MOM6-examples
git checkout *MOM6_GIT_TAG #753f135 this is just the value of MOM6_GIT_TAG property, can't seem to use variable
git submodule update --recursive --init src/MOM6 src/SIS2 src/icebergs

```

Comment (singhd789) [reply am5-compile/compile.yaml:115]: It looks like all of the `additional_instructions` are read in as a string with the use of `|`. If you did want to specify tags in the `fre_properties`, it looks like this format might work using the `!join` constructor:
```
          additionalInstructions: !join ["cd mom6\n",
                                         "git clone https://github.com/NOAA-GFDL/MOM6-examples.git -b dev/gfdl\n",
                                         "cd MOM6-examples\n",
                                         "git checkout 753f135\n",
                                         "git submodule update --recursive --init src/MOM6 src/SIS2 src/icebergs\n",
                                         "cd src/MOM6\n",
                                         "git checkout ",*MOM6_GIT_TAG,
                                         "cd ../SIS2\n",
                                         "git checkout 00c3b5be13721a7af99f62f2163969e8dd303df7\n",
                                         "cd ../icebergs\n",
                                         "git checkout 6a8944fc7544608942de9a42bd3b96f6e56684e0\n",
                                         "\ncd ../../../.."]

```

Comment (uramirez8707) [reply am5-compile/compile.yaml]: that works https://github.com/NOAA-GFDL/fre-examples/pull/1/commits/199169c3e158f561e11636fea6ec0915251fb029

**Pull request merged**